src/tests/WebApi.OpenAPI.SystemTest/openapi-v1.yaml

```diff
@@ -489,7 +474,7 @@ paths:
         '200':
           description: OK
           content:
-            application/json:
```

Comment:
This one is weird

Reply:
The controller is annotated explicitly with `text/plain`. The previous behavior, while correct, is also weird. If we consider this is important, maybe a new analyzer rule would be better.